db/db_impl/db_impl.cc

```diff
@@ -1872,7 +1885,8 @@ std::vector<Status> DBImpl::MultiGet(
       PERF_TIMER_GUARD(get_from_output_files_time);
       super_version->current->Get(read_options, lkey, &pinnable_val, timestamp,
                                   &s, &merge_context,
-                                  &max_covering_tombstone_seq);
+                                  &max_covering_tombstone_seq, nullptr, nullptr,
```

Comment:
Nit: it improves readability by doing sth. like `/*value_found=*/nullptr` for constants.

Reply:
I feel a modern IDE should help to do that.
For example, I'm using CLion, it's able to auto display arguments name if it's constant:
![image](https://user-images.githubusercontent.com/33968/93920587-3919c080-fcc4-11ea-86d5-831f98f4d294.png)

Reply:
The assumption of a modern IDE feels too strong.

Reply:
Updated.

Even though I'm fully convinced that it's a better style, but I don't have strong option about that either.